### ISSUE TITLE
There is No Possibility to Pass Any Args to svgop.exe

### ISSUE THREAD

**Issue by Antonytm** [twardoch/svgop#7]
I as user of `svgop.exe` want to disable one of plugins, let's say `removeViewBox`

But the list of plugins is pre-configured:
https://github.com/twardoch/svgop/blob/master/src/app/svgop-pkg.js

It makes custom usages of SVGOP quite hard. The only one option is to reconfigure sources and recompile them.

**Comment by twardoch** [comment]
True. I'm a copy-paste JS/C++ person, don't really know what I'm doing. It's certainly possible to add CLI args but I don't know how
